manager/ui/war/plugins/api-manager/html/api/api-impl.html

```diff
@@ -104,6 +104,19 @@
               </div>
             </div>
 
+            <div style="margin-top: 10px">
+              <input ng-model="updatedApi.disableKeysStrip"
+                     type="checkbox"
+                     id="disable-key-strip"
+                     ng-disabled="isEntityDisabled()">
+              <label for="disable-key-strip"
+                     apiman-i18n-key="disable-key-strip-msg"

```

Comment:
Do you have added this also in the `io/apiman/manager/ui/server/i18n/template_messages.properties`?

Reply:
Done!